config/routes.rb

```diff
@@ -16,6 +16,7 @@
   namespace :api do
     namespace :v1 do
       resources :recipe
+      resources :user
```

Comment:
I think technically, to adhere to rails conventions, we should pluralize this and use `users`. But I think this means we'll also need to pluralize `user_controller.rb` and `API::V1::UsersController`.

The reasoning is that it's kinda weird to hit the `api/v1/user` endpoint and get back an array of users.

Reply:
pluralized to users throughout